### PR TITLE
fix: a11y issues

### PR DESCRIPTION
**Pull request by theogainey**
### Description
fix TradingCardImage alt text
fix Logo hidden text
fix CurrentProjects Table to be labeled by section heading

### Spec

See Story: [236](https://sparkbox.atlassian.net/browse/FSA22V1-236)

### Validation

* [ ] This PR affects production code, so it was browser tested (see below).
* [ ] Changes are browser tested. This includes functionality, accessibility, responsiveness, and design.

#### To Validate

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, Snyk, etc).
2. Pull down all related branches.
3. Confirm all tests pass.

### CODE REVIEW

File: components/CurrentProjects.jsx [hunk 1]
```diff
@@ -34,14 +34,13 @@ export default function CurrentProjects({ currentProjects }) {
...
+      <h2 id="current-projects" className="cmp-current-projects__header cmp-current-projects__header--heading">Current Projects</h2>
       <Show when={!currentProjects.length}>
-        <h2 id="current-projects" className="cmp-current-projects__header cmp-current-projects__header--heading">Current Projects</h2>
         <p className="cmp-current-projects__status">This team member is not currently on a client project.</p>
       </Show>
 
       <Show when={currentProjects.length}>
-        <table className="cmp-current-projects__table">
-          <caption className="cmp-current-projects__header cmp-current-projects__header--caption">Current Projects</caption>
+        <table className="cmp-current-projects__table" aria-labelledby="current-projects">
```
Comment (dustin-jw) [inline components/CurrentProjects.jsx:43]: I hadn't seen the use of the `<Show>` component yet, I like it!

**Review by dustin-jw — DISMISSED** [review]
Nicely done! @arnest00 want to double check it?

File: components/TradingCardImage.jsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ export default function TradingCardImage({ imageUrl, weeklyCapacity, forecastedH
             src={imageUrl}
             height={350}
             width={350}
-            alt={imageUrl}
+            alt=""
```
[the remark this answers is not in the document]
Comment (arnest00) [reply components/TradingCardImage.jsx:15]: Ditto!

**Review by arnest00 — APPROVED** [review]

**Comment by arnest00** [comment]
Let me know when you rebase and I'll reapprove, @theogainey!

**Review by arnest00 — APPROVED** [review]

**Pull request merged**